beerlog/gui/display.py

```diff
@@ -358,8 +358,14 @@ def GetAchievements(self, name):
 
     return all_achievements
 
-  def _ShowAchievement(self, achievement):
-    """Displays an achievement"""
+  def _ShowAchievement(self, achievement, picture_path=None):
+    """Displays an achievement
+
+    Args:
+        achievement(Achivement): the achievement to display.
+        picture_path(str): if specified, will write the picture to the
```

Comment:
Use [mock](https://docs.python.org/3/library/unittest.mock.html) instead of adding this parameter.

Reply:
I split the function in 2 instead